### PR TITLE
Add translations for Persian, Igbo, Yoruba and Pidgin cookie banners

### LINKED ISSUE #1887: Add translations for the Cookie and Privacy banners for Igbo, Yoruba and Pidgin
**Is your feature request related to a problem? Please describe.**
When I see the privacy and/or cookie banners on Igbo, Yoruba or Pidgin, then I want to see them written in the correct language.

**Describe the solution you'd like**
Add the translations for these services from the spreadsheet.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #1887 & #1438 

**Overall change:** 

Translations added for four services for the privacy and cookie banners

**Code changes:**

- Replace English text with correct language text for four services
- Tweak quotemarks used to keep prettier happy

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by jamesbhobbs — DISMISSED** [review]
All the formatting looks good to me, I've no way of knowing if the strings are correct.

**Review by ghost — DISMISSED** [review]

File: src/app/lib/config/services/persian.js [hunk 1]
```diff
@@ -52,49 +52,50 @@ const persian = {
...
       },
       cookie: {
-        title: 'Let us know you agree to cookies',
+        title: 'آیا با دریافت کوکی‌های ما موافقید؟',
         description: {
           uk: {
-            first: 'We use ',
-            linkText: 'cookies',
+            first: ' ما برای بهبود استفاده شما از خدمات بی‌بی‌سی از',
+            linkText: 'کوکی',
```
Comment (sareh) [inline src/app/lib/config/services/persian.js:81]: The link text 'cookie' merges into the words following and preceding it, due to the lack of spaces
```suggestion
            linkText: ' کوکی ',
```

Edit for clarity for those not reading Persian - the orange link text is joined up to the grey words either side. It should have a space either side. 

Screen Shot cookie banner

File: src/app/lib/config/services/persian.js [hunk 2]
```diff
@@ -52,49 +52,50 @@ const persian = {
...
+              'استفاده می‌کنیم. اگر موافق هستید دریافت کوکی‌ها را تایید کنید ',
             linkUrl:
               'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
           },
           international: {
-            first: 'We and our partners use technologies, such as ',
-            linkText: 'cookies',
+            first:
+              ' ما و شریکانمان برای اینکه خدمات آنلاین بهتری به شما ارائه کنیم و محتوای مورد نظرتان را به شما نشان دهیم از تکنولوژی‌هایی مانند',
+            linkText: 'کوکی‌ها',
```
Comment (sareh) [inline src/app/lib/config/services/persian.js:90]: Likewise here - without the spaces, this word 'cookies' will join up to the word following it. 
```suggestion
            linkText: ' کوکی‌ها ',
```
Screen Shot cookie banner international

File: src/app/lib/config/services/persian.js (hunk 1 above)
Comment (jimjohnsonrollings) [reply src/app/lib/config/services/persian.js:81]: Moving the space from the left end of 'first' to the right end of 'first', and the opposite for 'last' seems to have fixed this.

Screen Shot 2019-06-11 at 12 13 23

File: src/app/lib/config/services/persian.js (hunk 2 above)
Comment (jimjohnsonrollings) [reply src/app/lib/config/services/persian.js:90]: As above, moved the spaces to the opposite ends of 'first' and 'last'

**Review by sareh — DISMISSED** [review]
👍 Looks great. Thanks for the fix, Jim.

**Review by ghost — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Comment by sareh** [comment]
The AMP cypress tests are timing out on Travis CI, though they're passing on Jenkins CI as well as locally. 

Screen Shot Travis

Screen Shot Jenkins

File: cypress/integration/cookieBannerWS.js [hunk 3]
```diff
@@ -13,8 +15,12 @@ describe('World Service Cookie Banner Test', () => {
         `${config[serviceName].assets.cookieAgreement}`,
       );
     });
+  });
+});
 
-    it(`should load the relevant translations for ${serviceName} (AMP)`, () => {
+describeForEuOnly('World Service Cookie Banner Transations - AMP', () => {
```
Comment (sareh) [inline cypress/integration/cookieBannerWS.js:21]: The issue was that AMP cookie banner will only show within the EU, so have added this describe block.  These tests will run on our Jenkins CI (within EU) and be skipped on Travis CI (outside of EU).

**Review by ghost — DISMISSED** [review]
Looks Great. 👍

File: data/prod/persian/articles/c4vlle3q337o.json [hunk 4]
```diff
@@ -7,9 +7,9 @@
     "type": "article",
     "createdBy": "",
     "created": 1535122662000,
-    "firstPublished": 1535122662000,
-    "lastPublished": 1535122662000,
-    "lastUpdated": 1535122662000,
+    "firstPublished": 1559914875000,
```
Comment (pjlee11) [inline data/prod/persian/articles/c4vlle3q337o.json]:
```suggestion
    "firstPublished": 1535122662000,
```

File: data/prod/persian/articles/c4vlle3q337o.json [hunk 5]
```diff
@@ -7,9 +7,9 @@
...
+    "lastPublished": 1559914880000,
```
Comment (pjlee11) [inline data/prod/persian/articles/c4vlle3q337o.json]:
```suggestion
    "lastPublished": 1535122662000,
```

File: data/prod/persian/articles/c4vlle3q337o.json [hunk 6]
```diff
@@ -7,9 +7,9 @@
...
+    "lastUpdated": 1559914875000,
```
Comment (pjlee11) [inline data/prod/persian/articles/c4vlle3q337o.json]:
```suggestion
    "lastUpdated": 1535122662000,
```

**Review by pjlee11 — APPROVED** [review]
LGTM 👍 

NB: I assisted James in writing the tests so ideally we want approvals

**Review by oluoluoxenfree — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**